docs/schema/reference.md

```diff
@@ -605,15 +611,15 @@ The items block is used to list the line-items associated with a tender, award o
 
 ### Milestone
 
-Milestone information can be included in the [planning](#planning), [tender](#tender), [contract](#contract) and [contract implementation](#implementation) blocks.
+Milestone information can be included in the [planning](#planning), [tender](#tender), [contract](#contract) and [contract implementation](#implementation) objects.
```

Comment:
```suggestion
Milestone information can be included in [planning](#planning), [tender](#tender), [contract](#contract) and [contract implementation](#implementation) objects.
```

Reply:
I think the 'the' is needed, if it was just one thing in the list "Milestone information can be included in planning object" wouldn't be right, it needs the "the"

Reply:
Hmm, I'm not a grammar expert, but if there was one thing in the list I think the sentence would be "Milestone information can be included in planning object**s**" because we're not talking about a specific instance of a planning object, rather we're talking about planning objects in general.

Reply:
It's fine either way, but I think it reads better without "the" because we're not trying to pick out specific objects.

Reply:
"the" removed in https://github.com/open-contracting/standard/pull/1660/commits/b8b7ca73ed0fb665048b1841ed287044447029fe